webapp/platform/types/src/reports.ts

```diff
@@ -18,6 +18,7 @@ export enum ReportSortDirection {
 }
 
 export enum ReportDuration {
+    AllTime = "all_time",
```

Comment:
Added so we have a clear moniker for the first option, despite it also being the default.